src/three-components/SmoothControls.ts

```diff
@@ -409,7 +412,12 @@ export class SmoothControls extends EventDispatcher {
     // the spherical->position derivation:
     if (!this[$previousPosition].equals(this.camera.position)) {
       this[$previousPosition].copy(this.camera.position);
-      this.dispatchEvent({type: 'change'});
+
+      const source = this[$isUserChange] ? 'user-interaction' : '';
```

Comment:
Let's give the other source a meaningful name. Maybe it's `'none'`, implying there was no special source? Or, we could say `author-configuration` to suggest that it originated from some author-provided configuration?

Reply:
Also, conventionally we set strings like this as module-wide `const` values. So towards the top of the module you would make a `const USER_INTERACTION_CHANGE_SOURCE = 'user-interaction';` and then use that here.